src/Display.tsx

```diff
@@ -171,22 +168,40 @@ const VictimDetail: React.FC<VictimDetailProps> = props => {
                 <FlexItem>Phone:</FlexItem>
                 <FlexItem>Needs First Aid:</FlexItem>
                 <FlexItem>Location:</FlexItem>
-                {props.data.status !== "REPORTED" && (
+                {props.data.status === status.reported && (
+                  <FlexItem>Neighbouring Location:</FlexItem>
+                )}
+                {props.data.status === status.assigned && (
+                  <FlexItem>Neighbouring Location:</FlexItem>
+                )}
+
+                {props.data.status !== status.reported && (
```

Comment:
It might be better to use inline if-else or ternary operator.
Something like: https://reactjs.org/docs/conditional-rendering.html

Reply:
Okay..